src/mocks/encryptor.mock.ts

```diff
@@ -1,10 +1,11 @@
-const sinon = require('sinon');
```

Comment:
If the mock files are exclusively for tests, what do you think about moving them back to `test`? Otherwise they will get written to `dist/` and included in releases accidentally.

Reply:
Agreed [352b11c](https://github.com/MetaMask/KeyringController/pull/202/commits/352b11cab967de2377dd6b7ac570218660126997)